code/datums/spells/horsemask.dm

```diff
@@ -26,7 +26,7 @@
 	if(!target)
 		return
 
-	if((target.type in compatible_mobs) || ishuman(target))
+	if((!target.type in compatible_mobs) || ishuman(target))

```

Comment:
this won't work regardless
you need the `!` behind the `(`
so like `!(target.type in compatible_mobs)`